pkg/chainaccessor/default_accessor.go

```diff
@@ -172,6 +174,7 @@ func (l *DefaultAccessor) MsgsBetweenSeqNums(
 		}
 
 		msg.Message.Header.OnRamp = onRampAddress
+		msg.Message.Header.TxHash = "0x" + hex.EncodeToString(item.TxHash)
```

Comment:
I suppose this pattern is already implemented as `Bytes.String()` function. It might be worth replacing it with `NewBytesFromString(item.TxHash).String()` or something similar

Reply:
Can we have a tiny unit test to cover that change?